# Using single-cell genomics to explore transcriptional divergence and cis-regulatory dynamics of duplicated genes 
Gene duplication is a major source of evolutionary innovation, enabling the emergence of novel expression patterns and functions. However, previous studies have primarily assessed expression divergence at the tissue level, often relying solely on differential expression as a metric. Leveraging single-cell genomics, we investigated the transcriptional dynamics and cis-regulatory evolution of duplicated genes in cultivated soybean (Glycine max), that has undergone two rounds of whole-genome duplication. Our analysis revealed extensive transcriptional divergence among these duplicated gene pairs both within and across tissues. These expression divergences are correlated with both sequence variation and the presence or absence of their associated accessible chromatin regions (ACRs), where cis-regulatory elements (CREs) reside. 

<img width="802" height="191" alt="image" src="https://github.com/user-attachments/assets/e0abb82e-7481-45a8-98a4-f94f6a294de7" />

We identified duplicated gene pairs and gene sets (derived from two rounds of whole-genome duplication in soybean that share the same ancestral gene). These were integrated with gene expression and chromatin accessibility profiles generated by scATAC-seq and snRNA-seq from a previous study. For each gene pair within each tissue, we analyzed expression correlation and differential expression levels to classify their expression patterns. We also used a BLAST-based approach to assess sequence variation in their associated ACRs and examined whether expression divergence is linked to ACR sequence differences. Additionally, we evaluated how expression patterns change across tissues and developmental stages and explored ACR dynamics underlying consistent or transitioning expression. Finally, we focused on the evolution of cell-type-specific expression among duplicated genes and identified candidate causal ACRs, which may offer valuable targets for precise trait manipulation through editing of specific CREs to improve crop traits.


